scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -284,60 +286,36 @@ def handle(self, line):
             tc.status = "failed"
 
 
-class PytestHarnessException(Exception):
-    """General exception for pytest."""
+class TwisterLibraryHarnessException(Exception):
+    """General exception for twister library harness usage."""
 
+class TwisterLibraryHarness(Harness, abc.ABC):
 
-class Pytest(Harness):
+    def _configure(self, instance: TestInstance, tool_name: str, report_filename: str, log_filename: str):
```

Comment:
why not `configure` (without underscore)? It is base method. Then `toolname`, `report_filename` and `log_filename` could be updated in `configure` methods in child classes